nipoppy/data/examples/sample_pipelines/rabies-0.5.1/descriptor-RABIES-0.5.1-preprocess.json

```
@@ -0,0 +1,587 @@
+            "name": "anat_template",
+            "description": "Anatomical file for the commonspace atlas.",
+            "optional": true,
+            "type": "String",
+            "value-key": "[ANAT_TEMPLATE]",
+            "command-line-flag": "--anat_template",
+            "default-value": "/home/rabies/.local/share/rabies/DSURQE_40micron_average.nii.gz"
+        },
+        {
+            "id": "brain_mask",
+            "name": "brain_mask",
+            "description": "Brain mask aligned with the template.",
+            "optional": true,
+            "type": "String",
+            "value-key": "[BRAIN_MASK]",
+            "command-line-flag": "--brain_mask",
+            "default-value": "/home/rabies/.local/share/rabies/DSURQE_40micron_mask.nii.gz"
+        },
+        {
+            "id": "WM_mask",
+            "name": "WM_mask",
+            "description": "White matter mask aligned with the template.",
+            "optional": true,
+            "type": "String",
+            "value-key": "[WM_MASK]",
+            "command-line-flag": "--WM_mask",
+            "default-value": "/home/rabies/.local/share/rabies/DSURQE_40micron_eroded_WM_mask.nii.gz"
+        },
+        {
+            "id": "CSF_mask",
+            "name": "CSF_mask",
+            "description": "CSF mask aligned with the template.",
+            "optional": true,
+            "type": "String",
+            "value-key": "[CSF_MASK]",
+            "command-line-flag": "--CSF_mask",
+            "default-value": "/home/rabies/.local/share/rabies/DSURQE_40micron_eroded_CSF_mask.nii.gz"
+        },
+        {
+            "id": "vascular_mask",
+            "name": "vascular_mask",
+            "description": "Can provide a mask of major blood vessels to compute associated nuisance timeseries. The default mask was generated by applying MELODIC ICA and selecting the resultingcomponent mapping onto major brain vessels.",
+            "optional": true,
+            "type": "String",
+            "value-key": "[VASCULAR_MASK]",
+            "command-line-flag": "--vascular_mask",
+            "default-value": "/home/rabies/.local/share/rabies/vascular_mask.nii.gz"
+        },
+        {
+            "id": "labels",
+            "name": "labels",
+            "description": "Labels file providing the atlas anatomical annotations.",
+            "optional": true,
+            "type": "String",
+            "value-key": "[LABELS]",
+            "command-line-flag": "--labels",
+            "default-value": "/home/rabies/.local/share/rabies/DSURQE_40micron_labels.nii.gz"
+        },
+        {
+            "id": "bids_dir",
+            "name": "bids_dir",
+            "description": "The root folder of a BIDS valid dataset (sub-XXXXX folders should be found at the top level in this folder).",
+            "optional": false,
+            "type": "String",
+            "value-key": "[BIDS_DIR]",
+            "default-value": "[[NIPOPPY_DPATH_BIDS]]"
```

Should put this in invocation instead
```suggestion
            "value-key": "[BIDS_DIR]"
```